src/material/form-field/directives/notched-outline.html

```diff
@@ -1,5 +1,5 @@
-<div class="mdc-notched-outline__leading"></div>
-<div class="mdc-notched-outline__notch" #notch>
+<div class="mat-mdc-notch-piece mdc-notched-outline__leading"></div>
```

Comment:
I introduced this class since there are a bunch of styles that target these 3 elements.